src/app/layout.tsx

```diff
@@ -46,6 +65,12 @@ export default async function RootLayout({
 }: {
   children: ReactNode;
 }) {
+  // @ts-ignore FIXME testing server-side metrics
+  const pathname = children?.props?.childProp?.segment;
+  logger.info("page_view", {
+    pathname,
+    NODE_ENV: process.env.NODE_ENV,
+  });
```

Comment:
BTW we should be able to replace `mozlog` with this, and it emitting events should continue to be this simple I think.